src/lexer.coffee

```diff
@@ -292,9 +292,13 @@ exports.Lexer = class Lexer
 
   # Matches JavaScript interpolated directly into the source via backticks.
   jsToken: ->
-    return 0 unless @chunk.charAt(0) is '`' and match = JSTOKEN.exec @chunk
-    @token 'JS', (script = match[0])[1...-1], 0, script.length
-    script.length
+    return 0 unless @chunk.charAt(0) is '`' and
+      (match = HERE_JSTOKEN.exec(@chunk) or JSTOKEN.exec(@chunk))
+    [js] = match
+    script = if js[0..2] is '```' then js[3...-3] else js[1...-1]
```

Comment:
If we start requiring backslashes to be escaped, that could break existing code. I think it's better to only convert `` \` ``, only within single-backtick blocks. That preserves backward compatibility, and gives people a way to output an escaped backtick (via a triple-backtick block).